ansible_rulebook/cli.py

```diff
@@ -87,7 +87,7 @@ def get_parser() -> argparse.ArgumentParser:
     parser.add_argument(
         "-S",
         "--source-dir",
-        help="Source dir",
+        help="Event source plugins dir",
```

Comment:
Maybe it is worth to mention that this is a local path intended for development. 
```suggestion
        help="Local event source plugins dir for development",
```